src/FlaUI.WebDriver/Services/ConditionParser.cs

```diff
@@ -31,19 +29,24 @@ public class ConditionParser : IConditionParser
         /// - Attribute equals attribute (e.g. `[name=value]`) not supported.
```

Comment:
Also update the comment above `Unicode escape characters or escape characters in the attribute name are not supported.` -> `Escape characters in the attribute name are not supported.`